packages/@webex/internal-plugin-metrics/src/metrics.js

```diff
@@ -69,23 +69,16 @@ const Metrics = WebexPlugin.extend({
       // eslint-disable-next-line no-undef
       domain:
         typeof window !== 'undefined' ? window.location.hostname || 'non-browser' : 'non-browser', // Check what else we could measure
-      client_id: this.webex.credentials.config.client_id,
-      user_id: this.webex.internal.device.userId,
```

Comment:
as we're removing `device.userId` so means not using `@webex/internal-plugin-device` package. So please remove this import from index.js file as well as this is causing a circular dependency with `@webex/internal-plugin-metrics` package.

Reply:
Thanks for pointing out. I have removed, @meta-dipanshu-sharma Please check.